src/components/BusinessCaseReview/GeneralRequestInfoReview/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import ReviewRow from 'components/ReviewRow';
+import {
+  DescriptionDefinition,
+  DescriptionList,
+  DescriptionTerm
+} from 'components/shared/DescriptionGroup';
+import { GeneralRequestInfoForm } from 'types/businessCase';
+
+type GeneralRequestInfoReviewProps = {
+  values: GeneralRequestInfoForm;
+};
+
+const GeneralRequestInfoReview = ({
+  values
+}: GeneralRequestInfoReviewProps) => {
+  return (
+    <DescriptionList title="General request information">
```

Comment:
I originally wanted to move this text into i18n since it's a new component. I opted not to because we are currently trying to figure out some weird naming issues between "request" and "project". It's not 100% clear how widespread that change will be, so I don't want to make any copy changes before that's resolved.